client/state/jetpack-connection-health/schema.js

```diff
@@ -0,0 +1,39 @@
+const requestError = {
+	type: [ 'string', 'null' ],
```

Comment:
This is currently only `string` right? I thought you intentionally stopped using `null` but I'm confused now.

Reply:
Switched to null, as discussed above! Sorry for the confusion!